examples/webpack/index.js

```diff
@@ -27,7 +27,7 @@ var app = express();
 // create the view engine with `react-engine`
 var engine = renderer.server.create({
   routes: require(path.join(__dirname + '/public/routes.jsx')),
-  routesFilePath: path.join(__dirname + '/public/routes.jsx')
+  reactRoutes: path.join(__dirname + '/public/routes.jsx')
```

Comment:
@masotime  the updated spec is here: https://github.com/paypal/react-engine#server-options-spec
`reactRoutes` existed in v1.x